docs/api/puppeteer.page.setextrahttpheaders.md

```diff
@@ -4,7 +4,19 @@ sidebar_label: Page.setExtraHTTPHeaders
 
 # Page.setExtraHTTPHeaders() method
 
-The extra HTTP headers will be sent with every request the page initiates. NOTE: All HTTP header names are lowercased. (HTTP headers are case-insensitive, so this shouldn’t impact your server code.) NOTE: page.setExtraHTTPHeaders does not guarantee the order of headers in the outgoing requests.
+The extra HTTP headers will be sent with every request the page initiates.
+
+:::tip
```

Comment:
is there a syntax that is both rendered by GitHub and by the website generator? 🤔

Reply:
We can use @remark.